app/models/business.rb

```diff
@@ -0,0 +1,13 @@
+class Business < ApplicationRecord
+  belongs_to :city
```

Comment:
Here it might be interesting to think about what makes a business truly unique. Could you combine the name with the address or something like that to ensure uniqueness?